template/deploy/helm/[[product]]-operator/README.md.j2

```diff
@@ -0,0 +1,46 @@
+[]: # =============
+[]: # This file is automatically generated from the templates in stackabletech/operator-templating
+[]: # DON'T MANUALLY EDIT THIS FILE
+[]: # =============
+
+# Helm Chart for Stackable Operator for {[ operator.pretty_string }]
+
+This Helm Chart can be used to install Custom Resource Definitions and the Operator for {[ operator.pretty_string }] provided by Stackable.
+
+
+## Requirements
+
+- Create a [Kubernetes Cluster with Stackable Nodes](../Readme.md)
+- Install [Helm](https://helm.sh/docs/intro/install/)
+
+
+## Install the Stackble Operator for {[ operator.pretty_string }]
+
+```bash
+helm install {[ operator.product_string }]-operator ./{[ operator.product_string }]-operator
```

Comment:
This doesn't work if executed in the folder where the README is located.

I think this README file is misplaced and now that I see the folder structure I think it's unnecesserily complicated. Why have another folder (`product-operator`) under the `helm` folder ?